lib/octicons_react/README.md

```diff
@@ -0,0 +1,200 @@
+# GitHub Octicons React Component
+
+[![npm version](https://img.shields.io/npm/v/%40github%2Focticons-react.svg)](https://www.npmjs.org/package/%40github%2Focticons-react)
+[![Build Status](https://travis-ci.org/primer/octicons.svg?branch=master)](https://travis-ci.org/primer/octicons)
+
+> [Octicons] for React
+
+## Install
+
+```
+$ npm install @github/octicons-react --save
+```
+
+## Usage
+
+### `<Octicon>`
+The `<Octicon>` component is really just the "shell" of an Octicon that renders
+the `<svg>` element and all of its attributes. To render a specific icon, you
+must pass it either via the `icon` prop, or as the only child:
+
+```jsx
+/**
+ * The prop form is shorter, but doesn't allow you to pass icon props.
+ */
+<Octicon icon={Icon} />
+
+/**
+ * The child form allows you to pass props.
+ */
+<Octicon><Icon x={10}/></Octicon>
```

Comment:
Is `x={10}` an actual api call? If not, I think we should change it to something like `size='large'` so it doesn't confuse anyone.

If there isn't any props you can pass right now to the child form. What do you think about dropping that example from the readme. I see the value in keeping it, but until we support props to manipulate the icon, there's not much point in promoting it.